Increase tests timeout to reduce flakiness

> [!NOTE]
> Incoming from:
> - https://github.com/FuelLabs/fuels-ts/issues/2967

# `launchNode`

This PR increased it from `5000` to `10000`, but it wasn't enough:
 - #3056

We can try with `15000`, and also add it to the failing test specifically instead of the whole suite / describe block.


1. #3040 —— [workflow](https://github.com/FuelLabs/fuels-ts/actions/runs/10582805141/job/29323363074#step:6:1033)
2. #3044 —— [workflow](https://github.com/FuelLabs/fuels-ts/actions/runs/10595681995/job/29361975956#step:6:1033)


```console
 FAIL  packages/account/src/test-utils/launchNode.test.ts > launchNode > should throw on error and log error message
Error: Test timed out in 5000ms.
```

# `contract-factory`

1. https://github.com/FuelLabs/fuels-ts/pull/2993 — [workflow](https://github.com/FuelLabs/fuels-ts/actions/runs/10692851573/job/29642023530?pr=2993#step:6:994)

```console
⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  packages/fuel-gauge/src/contract-factory.test.ts > Contract Factory > deploys large contracts via blobs and awaits transaction id
Error: Test timed out in 5000ms.
If this is a long-running test, pass a timeout value as the last argument or configure it globally with "testTimeout".
⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯

 Test Files  1 failed | 376 passed (377)
      Tests  1 failed | 2196 passed | 4 skipped (2201)
   Start at  00:34:58
   Duration  178.41s (transform 77.37s, setup 135ms, collect 1108.61s, tests 975.44s, environment 337ms, prepare 273.12s)

 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

--- a/.changeset/pink-apricots-sell.md
+++ b/.changeset/pink-apricots-sell.md
@@ -1,0 +1,4 @@
+---
+---
+
+chore: fixing flaky tests

--- a/packages/account/src/test-utils/launchNode.test.ts
+++ b/packages/account/src/test-utils/launchNode.test.ts
@@ -28,7 +28,7 @@ vi.mock('fs', async () => {
 /**
  * @group node
  */
-describe('launchNode', { timeout: 10000 }, () => {
+describe('launchNode', () => {
   test('using ephemeral port 0 is possible', async () => {
     const { cleanup, port, url } = await launchNode({ port: '0' });
     expect(await fetch(url)).toBeTruthy();
@@ -116,7 +116,7 @@ describe('launchNode', { timeout: 10000 }, () => {
     cleanup();
   });
 
-  test('should throw on error and log error message', async () => {
+  test('should throw on error and log error message', { timeout: 15000 }, async () => {
     const logSpy = vi.spyOn(console, 'log');
 
     const invalidCoin = {

--- a/packages/fuel-gauge/src/contract-factory.test.ts
+++ b/packages/fuel-gauge/src/contract-factory.test.ts
@@ -292,33 +292,37 @@ describe('Contract Factory', () => {
     expect(value.toNumber()).toBe(1001);
   }, 15000);
 
-  it('deploys large contracts via blobs and awaits transaction id', async () => {
-    using launched = await launchTestNode({
-      nodeOptions: {
-        args: ['--tx-pool-ttl', '1s'],
-      },
-      providerOptions: {
-        resourceCacheTTL: -1,
-      },
-    });
+  it(
+    'deploys large contracts via blobs and awaits transaction id',
+    { timeout: 15000 },
+    async () => {
+      using launched = await launchTestNode({
+        nodeOptions: {
+          args: ['--tx-pool-ttl', '1s'],
+        },
+        providerOptions: {
+          resourceCacheTTL: -1,
+        },
+      });
 
-    const {
-      wallets: [wallet],
-    } = launched;
+      const {
+        wallets: [wallet],
+      } = launched;
 
-    const factory = new ContractFactory(LargeContractFactory.bytecode, LargeContract.abi, wallet);
-    const deploy = await factory.deployAsBlobTx<LargeContract>();
-    const initTxId = deploy.waitForTransactionId();
-    expect(initTxId).toStrictEqual(new Promise(() => {}));
-    const { contract } = await deploy.waitForResult();
-    expect(contract.id).toBeDefined();
-    const awaitTxId = await deploy.waitForTransactionId();
-    expect(awaitTxId).toBeTruthy();
+      const factory = new ContractFactory(LargeContractFactory.bytecode, LargeContract.abi, wallet);
+      const deploy = await factory.deployAsBlobTx<LargeContract>();
+      const initTxId = deploy.waitForTransactionId();
+      expect(initTxId).toStrictEqual(new Promise(() => {}));
+      const { contract } = await deploy.waitForResult();
+      expect(contract.id).toBeDefined();
+      const awaitTxId = await deploy.waitForTransactionId();
+      expect(awaitTxId).toBeTruthy();
 
-    const call = await contract.functions.something().call();
-    const { value } = await call.waitForResult();
-    expect(value.toNumber()).toBe(1001);
-  });
+      const call = await contract.functions.something().call();
+      const { value } = await call.waitForResult();
+      expect(value.toNumber()).toBe(1001);
+    }
+  );
 
   it('deploys large contracts via blobs [padded]', async () => {
     using launched = await launchTestNode({

